### PR TITLE
Makes the 556 and 762 calibers more appropriate to their naming

### PR DESCRIPTION
**Pull request by Anewbe**
Long time coming, shouldn't have any effect on their actual effectiveness, but now the guns that hit hard say they have a bigger bullet than the ones that hit less hard.

EDIT BECAUSE I'M A TERRIBLE DOCUMENTOR: The bullets that used to do 25 damage were called 762 rounds. These will still do 25 damage, but will be called 556 rounds. The bullets that used to do 35 damage were called 556 rounds. These will still do 35 damage, but will now be called 762 rounds. It looks a lot more complex than it actually is, which is unfortunate.

### CODE REVIEW

File: code/modules/projectiles/guns/projectile/automatic.dm [hunk 1]
```diff
@@ -350,12 +350,12 @@
 	item_state = "bullpup"
 	w_class = ITEMSIZE_LARGE
 	force = 10
-	caliber = "a556"
+	caliber = "a762"
```
Comment (SpadesNeil) [inline code/modules/projectiles/guns/projectile/automatic.dm]: According to the description, this still uses 5.56 ammo. This is not correct.

Or, personally, I think it'd be better if we switched the Space AK for this thing. Don't change the STS ammo or this Assault Carbine's ammo. Just switch which ones are available to players. This gun being a 7.62 makes no sense anyway, and it doesn't match the sprites provided.

File: code/modules/projectiles/guns/projectile/boltaction.dm [hunk 2]
```diff
@@ -2,29 +2,29 @@
 
 /obj/item/weapon/gun/projectile/shotgun/pump/rifle
 	name = "bolt action rifle"
-	desc = "A reproduction of an almost ancient weapon design from the early 20th century. It's still popular among hunters and collectors due to its reliability. Uses 7.62mm rounds."
+	desc = "A reproduction of an almost ancient weapon design from the early 20th century. It's still popular among hunters and collectors due to its reliability. Uses 5.56mm rounds."
```
Comment (SpadesNeil) [inline code/modules/projectiles/guns/projectile/boltaction.dm]: This is a terrible idea and I don't know why you're doing this. The whole point of the bolt action rifle is it has the same ROF as a shotgun, but is easier to reload in exchange for having a much shittier magazine. If anything, this deserves to stay as 7.62 even with the ammo's buff. It makes the weapon a lot less useless.

This also makes no sense considering the only gun available to players that is a bolt rifle are either the Mosins from cargo, or the competition shooting rifles in security.

To balance this, if you think it's too easy to get Mosins via cargo, adjust the point cost, or adjust how many you get with each crate.

In all, I am strongly against changing the bolt action rifle to 5.56, even with the buff to damage. It makes no sense to lore or balance.

File: code/modules/projectiles/guns/projectile/boltaction.dm [hunk 3]
```diff
@@ -2,29 +2,29 @@
...
-	ammo_type = /obj/item/ammo_casing/a762/blank
+	desc = "A bolt-action rifle with a heavy, high-quality wood stock that has a beautiful finish. Clearly not intended to be used in combat. Uses 5.56mm rounds."
+	ammo_type = /obj/item/ammo_casing/a556/blank
 
 /obj/item/weapon/gun/projectile/shotgun/pump/rifle/mosin
 	name = "\improper Mosin Nagant"
-	desc = "Despite its age, the Mosin Nagant continues to be a favorite weapon among colonists, conscripts, and militias across the cosmos. Most today are built by Chen-Iltchenko Firearms, but it's hard to say who built this particular gun, considering the design has been ripped off by just about every arms manufacturer in the galaxy. Uses 7.62mm rounds."
+	desc = "Despite its age, the Mosin Nagant continues to be a favorite weapon among colonists, conscripts, and militias across the cosmos. Most today are built by Chen-Iltchenko Firearms, but it's hard to say who built this particular gun, considering the design has been ripped off by just about every arms manufacturer in the galaxy. Uses 5.56mm rounds."
```
Comment (SpadesNeil) [inline code/modules/projectiles/guns/projectile/boltaction.dm]: Mosin is not filthy capitalist 5.56 weapon

File: code/modules/projectiles/guns/projectile/boltaction.dm [hunk 4]
```diff
@@ -55,13 +55,13 @@
 //Lever actions are the same thing, but bigger.
 /obj/item/weapon/gun/projectile/shotgun/pump/rifle/lever
 	name = "lever-action rifle"
-	desc = "A reproduction of an almost ancient weapon design from the 19th century. This one uses a lever-action to move new rounds into the chamber. Uses 5.56mm rounds."
+	desc = "A reproduction of an almost ancient weapon design from the 19th century. This one uses a lever-action to move new rounds into the chamber. Uses 7.62mm rounds."
```
Comment (SpadesNeil) [inline code/modules/projectiles/guns/projectile/boltaction.dm]: Again, this doesn't make sense at all.

File: code/modules/projectiles/guns/projectile/boltaction.dm [hunk 5]
```diff
@@ -55,13 +55,13 @@
...
 	item_state = "leveraction"
 	icon_state = "leveraction"
 	fire_sound = 'sound/weapons/rifleshot.ogg'
 	max_shells = 5
-	caliber = "a556"
+	caliber = "a762"
 	origin_tech = list(TECH_COMBAT = 1)// Old as shit rifle doesn't have very good tech.
-	ammo_type = /obj/item/ammo_casing/a556
+	ammo_type = /obj/item/ammo_casing/a762
 	load_method = SINGLE_CASING|SPEEDLOADER
```
Comment (SpadesNeil) [inline code/modules/projectiles/guns/projectile/boltaction.dm]: In fact, for realism/believably reasons, a speed loader should not work with this gun. This looks like a bad copypaste of a weapon with a new sprite.

Honestly, this should probably fire .357 ammo anyway. Basically a glorified shotgun with a smaller mag size and no special ammo in terms of damage dealt.

File: code/modules/projectiles/projectile/bullets.dm [hunk 6]
```diff
@@ -198,26 +198,26 @@
 	penetrating = 1
 
 /obj/item/projectile/bullet/rifle/a762
-	damage = 25
+	damage = 35
```
Comment (SpadesNeil) [inline code/modules/projectiles/projectile/bullets.dm]: This is good. I like this.

File: code/modules/projectiles/projectile/bullets.dm [hunk 7]
```diff
@@ -198,26 +198,26 @@
...
 
 /obj/item/projectile/bullet/rifle/a762/ap
-	damage = 20
-	armor_penetration = 50 // At 40 or more armor, this will do more damage than standard rounds.
+	damage = 30
+	armor_penetration = 50 // At 30 or more armor, this will do more damage than standard rounds.
 
 /obj/item/projectile/bullet/rifle/a762/hollow
-	damage = 35
+	damage = 4
```
Comment (SpadesNeil) [inline code/modules/projectiles/projectile/bullets.dm]: wait what this looks like a typo

File: code/modules/projectiles/projectile/bullets.dm [hunk 8]
```diff
@@ -198,26 +198,26 @@
...
 	armor_penetration = -50
 	penetrating = 0
 
 /obj/item/projectile/bullet/rifle/a556
-	damage = 35
+	damage = 25
```
Comment (SpadesNeil) [inline code/modules/projectiles/projectile/bullets.dm]: This is good. I like this.

File: code/modules/projectiles/projectile/bullets.dm [hunk 9]
```diff
@@ -198,26 +198,26 @@
...
 
 /obj/item/projectile/bullet/rifle/a556/ap
-	damage = 30
-	armor_penetration = 50 // At 30 or more armor, this will do more damage than standard rounds.
+	damage = 20
+	armor_penetration = 50 // At 40 or more armor, this will do more damage than standard rounds.
 
 /obj/item/projectile/bullet/rifle/a556/hollow
-	damage = 40
+	damage = 35
```
Comment (SpadesNeil) [inline code/modules/projectiles/projectile/bullets.dm]: Yea that was absolutely a typo earlier.

File: code/modules/random_map/drop/drop_types.dm [hunk 10]
```diff
@@ -83,8 +83,8 @@ var/global/list/datum/supply_drop_loot/supply_drop
 		/obj/item/clothing/accessory/storage/black_drop_pouches,
 		/obj/item/weapon/storage/backpack/dufflebag/sec,
 		/obj/item/weapon/gun/projectile/automatic/carbine,
-		/obj/item/ammo_magazine/a556m/ap,
-		/obj/item/ammo_magazine/a556m,
+		/obj/item/ammo_magazine/c762/ap,
+		/obj/item/ammo_magazine/c762,
```
Comment (SpadesNeil) [inline code/modules/random_map/drop/drop_types.dm]: If you take my earlier suggestion to leave the STS and Carbine as they are, and just switch which ones are available, then remember to put these back to the way they were.

File: maps/northern_star/polaris-2.dmm [hunk 11]
```diff
@@ -869,11 +869,11 @@
 "aqK" = (/obj/machinery/door/blast/regular{icon_state = "pdoor1"; id = "ASSAULT"; name = "Assault Weapon Storage"; p_open = 0},/turf/unsimulated/floor{icon_state = "dark"},/area/centcom/specops)
 "aqL" = (/obj/structure/sign/securearea,/turf/simulated/shuttle/wall/dark/hard_corner,/area/centcom/specops)
 "aqM" = (/obj/machinery/door/airlock/centcom,/turf/unsimulated/floor{icon_state = "dark"},/area/centcom/specops)
-"aqN" = (/obj/structure/table/rack,/obj/item/ammo_magazine/a556,/obj/item/ammo_magazine/a556,/obj/item/ammo_magazine/a556,/obj/item/ammo_magazine/a556,/obj/item/ammo_magazine/a556,/obj/item/ammo_magazine/a556,/obj/item/ammo_magazine/a556,/obj/item/ammo_magazine/a556,/turf/unsimulated/floor{icon_state = "dark"},/area/centcom/specops)
+"aqN" = (/obj/structure/table/rack,/obj/item/ammo_magazine/c762,/obj/item/ammo_magazine/c762,/obj/item/ammo_magazine/c762,/obj/item/ammo_magazine/c762,/obj/item/ammo_magazine/c762,/obj/item/ammo_magazine/c762,/obj/item/ammo_magazine/c762,/obj/item/ammo_magazine/c762,/turf/unsimulated/floor{icon_state = "dark"},/area/centcom/specops)
```
Comment (SpadesNeil) [inline maps/northern_star/polaris-2.dmm]: Ditto.

File: maps/northern_star/polaris-2.dmm [hunk 12]
```diff
@@ -869,11 +869,11 @@
...
 "aqO" = (/obj/structure/table/rack,/obj/item/weapon/gun/projectile/automatic/z8,/obj/item/weapon/gun/projectile/automatic/z8,/turf/unsimulated/floor{icon_state = "dark"},/area/centcom/specops)
 "aqP" = (/obj/structure/table/rack,/obj/item/weapon/gun/energy/sniperrifle,/turf/unsimulated/floor{icon_state = "dark"},/area/centcom/specops)
 "aqQ" = (/obj/structure/table/rack,/obj/item/ammo_magazine/p90,/obj/item/ammo_magazine/p90,/obj/item/ammo_magazine/p90,/obj/item/ammo_magazine/p90,/obj/item/weapon/gun/projectile/automatic/p90,/obj/item/weapon/gun/projectile/automatic/p90,/turf/unsimulated/floor{icon_state = "dark"},/area/centcom/specops)
-"aqR" = (/obj/structure/table/rack,/obj/item/ammo_magazine/a762,/obj/item/ammo_magazine/a762,/obj/item/ammo_magazine/a762,/obj/item/ammo_magazine/a762,/obj/item/weapon/gun/projectile/automatic/l6_saw,/turf/unsimulated/floor{icon_state = "dark"},/area/centcom/specops)
+"aqR" = (/obj/structure/table/rack,/obj/item/ammo_magazine/a556,/obj/item/ammo_magazine/a556,/obj/item/ammo_magazine/a556,/obj/item/ammo_magazine/a556,/obj/item/weapon/gun/projectile/automatic/l6_saw,/turf/unsimulated/floor{icon_state = "dark"},/area/centcom/specops)
```
Comment (SpadesNeil) [inline maps/northern_star/polaris-2.dmm]: Ditto.

**Comment by SpadesNeil** [comment]
Also the SAW can stay 5.56, tbh.

Making it 7.62 is just cruel.

File: code/modules/projectiles/guns/projectile/boltaction.dm (hunk 3 above)
Comment (Anewbe) [inline code/modules/projectiles/guns/projectile/boltaction.dm]: Does no change to balance, keeps it in line with the rest of the pump rifles.

File: code/modules/projectiles/guns/projectile/boltaction.dm (hunk 2 above)
Comment (Anewbe) [inline code/modules/projectiles/guns/projectile/boltaction.dm]: Does no change to balance.

File: code/modules/projectiles/guns/projectile/automatic.dm (hunk 1 above)
Comment (Anewbe) [inline code/modules/projectiles/guns/projectile/automatic.dm]: Missed a spot, but no, I do not have the sprites to replace the STS with the bullpup, nor do I particularly want to. The sprites have been changed to work as they do now.

File: code/modules/projectiles/guns/projectile/boltaction.dm (hunk 4 above)
Comment (Anewbe) [inline code/modules/projectiles/guns/projectile/boltaction.dm]: There have been talks of adding that sort of round, but it's outside the scope of this PR.

File: code/modules/projectiles/guns/projectile/boltaction.dm (hunk 5 above)
Comment (Anewbe) [inline code/modules/projectiles/guns/projectile/boltaction.dm]: Speedloader maybe, .357 maybe. Both outside the scope of this PR.

File: code/modules/projectiles/projectile/bullets.dm (hunk 7 above)
Comment (Anewbe) [inline code/modules/projectiles/projectile/bullets.dm]: Typo, will be fixed.

**Comment by SpadesNeil** [comment]
Anyway, you've heard all my ramblings on Discord. I say this is otherwise fine to merge as it is. I can always make the tweaks I talked about afterward, and then everyone can have an input.

However, this needs to be accepted and merged before I can consider those tweaks.

File: code/modules/projectiles/guns/projectile/boltaction.dm (hunk 5 above)
Comment (SpadesNeil) [inline code/modules/projectiles/guns/projectile/boltaction.dm]: Fair enough. I'm otherwise okay with this PR. We can always make further adjustments in a different PR.

**Comment by SpadesNeil** [comment]
So I take it you never saw the reply I made in #3432.

**Comment by Anewbe** [comment]
Didn't notice it, but your PR changes far more than this one, and I'm not going to merge it for that reason.

**Comment by SpadesNeil** [comment]
Your PR: +130 −140
My PR: +33 −33

I have no idea what you're talking about.

**Comment by Anewbe** [comment]
Yours changes content, mine changes semantics.

**Comment by SpadesNeil** [comment]
That's accurate, I'll give you that.

**Comment by MagmaRam** [comment]
I'm merging this as a PR that purely changes the names, changes to what guns use which ammo can/will /should come in a different PR.

File: code/modules/projectiles/ammunition/boxes.dm [hunk 13]
```diff
@@ -375,33 +388,14 @@
...
 	name = "magazine (7.62mm)"
-	icon_state = "c762"
+	icon_state = "c7.62"
 	mag_type = MAGAZINE
 	caliber = "a762"
-	matter = list(DEFAULT_WALL_MATERIAL = 4000)
+	matter = list(DEFAULT_WALL_MATERIAL = 2000)
 	ammo_type = /obj/item/ammo_casing/a762
-	max_ammo = 20
+	max_ammo = 10
```
Comment (SpadesNeil) [inline code/modules/projectiles/ammunition/boxes.dm]: I just realized. Why?

File: code/modules/projectiles/ammunition/boxes.dm [hunk 14]
```diff
@@ -416,7 +410,7 @@
 	icon_state = "SVD"
 	mag_type = MAGAZINE
 	caliber = "a762"
-	matter = list(DEFAULT_WALL_MATERIAL = 2000)
+	matter = list(DEFAULT_WALL_MATERIAL = 1000)
 	ammo_type = /obj/item/ammo_casing/a762
 	max_ammo = 10
```
Comment (SpadesNeil) [inline code/modules/projectiles/ammunition/boxes.dm]: This is already a thing that exists.

File: code/modules/projectiles/guns/projectile/automatic.dm [hunk 15]
```diff
@@ -346,16 +346,16 @@
...
 	item_state = "bullpup"
 	w_class = ITEMSIZE_LARGE
 	force = 10
-	caliber = "a556"
+	caliber = "a762"
 	origin_tech = list(TECH_COMBAT = 6, TECH_MATERIAL = 1, TECH_ILLEGAL = 4)
 	slot_flags = SLOT_BACK
 	load_method = MAGAZINE
-	magazine_type = /obj/item/ammo_magazine/a556m
-	allowed_magazines = list(/obj/item/ammo_magazine/a556, /obj/item/ammo_magazine/a556m)
```
Comment (SpadesNeil) [inline code/modules/projectiles/guns/projectile/automatic.dm]: You also broke this gun's ability to hold smaller or larger magazines entirely. On the one hand you pulled it out of the admin abuse pit but on the other hand you also made the icons completely inconsistent.

File: code/modules/projectiles/ammunition/boxes.dm [hunk 16]
```diff
@@ -286,49 +286,43 @@
...
 	ammo_type = /obj/item/ammo_casing/a556
-	max_ammo = 10
+	max_ammo = 20
 	multiple_sprites = 1
 
-/obj/item/ammo_magazine/a556/empty
+/obj/item/ammo_magazine/c556/ext
+	name = "extended magazine (5.56mm)"
+	matter = list(DEFAULT_WALL_MATERIAL = 2700)
+	max_ammo = 30
```
Comment (SpadesNeil) [inline code/modules/projectiles/ammunition/boxes.dm]: Why do THESE exist?
